webapp/src/main/resources/templates/frontend/blog/future-plans.html

```diff
@@ -0,0 +1,84 @@
+<body>
+    <section layout:fragment="content">
+        <article id="blog-content-wrapper" class="mt-4 d-flex justify-content-center">
+            <div class="blog-content">
+                <h1 class="h4">Future plans</h1>
+                <div id="blog-author-begin" class="blog-post-author d-flex align-items-center mt-4">
+                    <img class="avatar avatar-s" src="#" th:src="@{/images/dummy-profile-picture.jpg}" alt="Avatar">
+                    <div class="ml-4 text-left">
+                        <p class="mb-0">DanielW</p>
+                        <p class="text-muted mt-n1 mb-0">February 1st, 2021</p>
+                    </div>
+                </div>
+
+                <div class="blog-text text-justify mt-4">
+                    <p>
+                        Since about a month the Metal Detector has opened its doors for a small selection of people. The development has
+                        not stood still since then and will not stand still the rest of the year. We still have plenty of features and
+                        improvements planned for the Metal Detector. Today I want to give you a small overview of what you can look
+                        forward to in the future.
+                    </p>
+
+                    <h2 class="h5 mt-4">Notifications</h2>
+                    <p>
+                        Currently the concept of the site is that you have to actively visit it and inform yourself when which band is
+                        releasing a new album. By restricting to your favorite bands, you are quickly up to date, but this manual effort is
+                        of course uncomfortable. Exactly for this reason there will soon be a notification system that will notify you
+                        proactively. For the time being this will be done via email and you can choose whether you want to receive a
+                        summary about recently released and upcoming album releases of your favorite band and/or additionally be notified
+                        on the day of announcement and/or release. In addition to e-email, channels such as Telegram or RSS feed are also
+                        conceivable in the future.
+                    </p>
+
+                    <h2 class="h5 mt-4">Desktop first, mobile second</h2>
+                    <p>
+                        One or the other will have noticed it already, but mobile the Metal Detector is rather a disaster and not really
+                        usable. Of course this does not fit to today's and we want to change that as soon as possible, because we want to
+                        be available and usable for you on the go. A big construction site this year will therefore be the optimization of
+                        the website for mobile devices. Currently there are no plans to release the Metal Detector as a native app for
+                        Android and iPhone. We definitely lack the manpower for that at the moment.
```

Comment:
```suggestion
                        Android and iPhone. We definitely lack the (wo)manpower for that at the moment.
```
For the fun of it? :)

Reply:
only womanpower :dancing_women: !